expression/expression.go

```diff
@@ -1184,6 +1191,9 @@ func canScalarFuncPushDown(scalarFunc *ScalarFunction, pc PbConverter, storeType
 
 func canExprPushDown(expr Expression, pc PbConverter, storeType kv.StoreType) bool {
 	if storeType == kv.TiFlash && expr.GetType().Tp == mysql.TypeDuration {
+		if pc.sc.InExplainStmt {
```

Comment:
duration and  func() will rusult two warnings? if func() cannot be pushed down

Reply:
If the `expr` contains `Duration`, it will return directly without checking `function` in this `expr`